Tanga/Features/DeleteAccount/DeleteAccountView.swift

```diff
@@ -0,0 +1,107 @@
+//
+//  DeleteAccountView.swift
+//  Tanga
+//
+//  Created by Rygel Louv on 11/02/2025.
+//
+
+import SwiftUI
+
+struct DeleteAccountView: View {
+    @Binding var path: NavigationPath
+    @State private var showDeleteAlert = false
+    @State private var showDeleteErrorAlert = false
+    @EnvironmentObject var authManager: AuthManager
+    @Environment(\.dismiss) private var dismiss
+    
+    var body: some View {
+        GeometryReader { geometry in
+            ScrollView {
+                VStack(spacing: 0) {
+                    // Content VStack
+                    VStack {
+                        DeletionImage()
+                        
+                        Text("Delete Your Tanga Account")
+                            .fontWeight(.bold)
+                            .font(Font.custom("Montserrat", size: 22, relativeTo: .title))
+                            .foregroundColor(.red)
+                            .multilineTextAlignment(.center)
+                            .padding(.top, 14)
+                            .padding(.bottom, 24)
+                            .padding(.horizontal, 12)
+                        
+                        Text(
+                            """
+                            Are you sure you want to say goodbye to your Tanga account? This action is permanent and cannot be undone. Here's what you'll miss out on:\n
+                                - All your saved book summaries\n
+                                - Any premium content or features you purchased \n
+                            If you're sure you want to go, click "Delete Account" below. If you have any questions or need a hand, our support team is here to help!
+                            """)
+                            .font(Font.custom("Montserrat", size: 14, relativeTo: .body))
+                            .fontWeight(.regular)
+                            .foregroundColor(.auroMetalSaurus)
+                    }
+                    
+                    // Spacer that pushes buttons to bottom
+                    Spacer(minLength: 0)
+                    
+                    // Buttons VStack
+                    VStack(spacing: 16) {
+                        TangaButton(
+                            onButtonTap: {
+                                dismiss()
+                            },
+                            text: "Keep My Account"
+                        )
```

Comment:
Another syntactic sugar 😅
```suggestion
                        TangaButton(
                            onButtonTap: dismiss.callAsFunction,
                            text: "Keep My Account"
                        )
```